git_machete/cli.py

```diff
@@ -457,7 +457,7 @@ def set_utils_global_variables(
 
 def get_branch_arg_or_current_branch(
         cli_opts: git_machete.options.CommandLineOptions, git_context: GitContext) -> LocalBranchShortName:
-    return cli_opts.opt_branch or git_context.get_current_branch()
+    return LocalBranchShortName.of(cli_opts.opt_branch) or git_context.get_current_branch()
```

Comment:
Hmm does it address other uses of `cli_opts.opt_branch` across the code (if any)? shouldn't the conversion to `LocalBranchShortName` happen rather in the place where `opt_branch` is assigned?

Reply:
Yup, switched to typing in `update_cli_opts_using_parsed_args`, which I believe is more precise action.